tests/modules/redshift_datasets/test_unit_redshift_connection_service.py

```diff
@@ -207,10 +189,10 @@ def test_list_environment_redshift_connections_unauthorized(
     ).contains('UnauthorizedOperation', 'LIST_ENVIRONMENT_REDSHIFT_CONNECTIONS', env_fixture.environmentUri)
 
 
-def test_list_connection_schemas(connection1_serverless, api_context_1, patch_redshift):
+def test_list_connection_schemas(connection1_serverless, api_context_1, patch_redshift_data):
     # When
     response = RedshiftConnectionService.list_connection_schemas(uri=connection1_serverless.connectionUri)
-    assert_that(response).is_equal_to(MockRedshiftDataClient().list_redshift_schemas())
+    assert_that(response).contains('public', 'dev')
 
 
 def test_list_connection_schemas_unauthorized(connection1_serverless, api_context_2):
```

Comment:
`connection1_serverless` is setting `api_context_2` but you also use the `api_context_2` fixture. How does that work? Is there a a conflict?

Reply:
What we need:
- create connection1 with api context 1
- list connections with api context 2
What happens:
- we use fixture connection1 - sets api context 1 and creates connection
- we use fixture api_context 2 - sets api context 2
- execute test code - list connections

If we defined the test as `
def test_list_connection_schemas_unauthorized(api_context_2, connection1_serverless, ):` then it would fail. 

I am not the biggest fun of it, but unless we are able to keep to api contexts in parallel I am not sure I know how to fix it